hotel-management/src/components/RoomCard/RoomCardPreview.tsx

```diff
@@ -2,27 +2,20 @@ import { Link } from 'react-router-dom';
 
 import DeleteIcon from '@/assets/images/delete-x.svg?react';
 import EditIcon from '@/assets/images/edit.svg?react';
+import { ROUTES } from '@/constants';
 
 import { Button } from '../Button';
 import { Image } from '../Image';
-
 import './index.css';
 
-type RoomCardPreviewProps = {
-  roomId: string;
-  price: number;
-  src: string;
-  onDelete: () => void;
-  onEdit: () => void;
-};
-
```

Comment:
Why we delete this?

Reply:
because it is defined in another file, this type is duplicated